doc/source/tutorial/tutorial_presenting_larray_objects.ipyml

```diff
@@ -1,492 +0,0 @@
-            and when the method `copy()` is called. Do not add metadata to an array if you know
-            you will apply actions or methods on it before dumping it.</li>
-      </ul>
-    
-    </div>
-
-
-- markdown: |
-    ### Array creation functions
-    
-    Arrays can also be generated in an easier way through [creation functions](../api.rst#array-creation-functions):
-    
-    -  `ndtest` : creates a test array with increasing numbers as data
-    -  `empty` : creates an array but leaves its allocated memory
-       unchanged (i.e., it contains "garbage". Be careful !)
-    -  `zeros`: fills an array with 0
-    -  `ones` : fills an array with 1
-    -  `full` : fills an array with a given value
-    -  `sequence` : creates an array from an axis by iteratively applying a function to a given initial value.
-    
-    Except for ndtest, a list of axes must be provided.
-    Axes can be passed in different ways:
-    
-    -  as Axis objects
-    -  as integers defining the lengths of auto-generated wildcard axes
-    -  as a string : 'gender=M,F;time=2007,2008,2009' (name is optional)
-    -  as pairs (name, labels)
-    
-    Optionally, the type of data stored by the array can be specified using argument dtype.
-
-
-- code: |
-    # start defines the starting value of data
-    ndtest((3, 3), start=-1)
-
-
-- code: |
-    # start defines the starting value of data
-    # label_start defines the starting index of labels
-    ndtest((3, 3), start=-1, label_start=2)
-
-
-- code: |
-    # empty generates uninitialised array with correct axes
-    # (much faster but use with care!).
-    # This not really random either, it just reuses a portion
-    # of memory that is available, with whatever content is there.
-    # Use it only if performance matters and make sure all data
-    # will be overridden.
-    empty([age, gender, time])
-
-
-- code: |
-    zeros([age, gender, time])
-
-
-- code: |
-    # dtype=int forces to store int data instead of default float
-    ones([age, gender, time], dtype=int)
-
-
-- code: |
-    full([age, gender, time], fill_value=1.23)
-
-
-- markdown: |
-    All the above functions exist in *(func)_like* variants which take axes from another array
-
-
-- code: |
-    ones_like(arr)
-
-
-- markdown: |
-    Create an array using the special ``sequence`` function (see link to documention of ``sequence`` in API reference for more examples):
-
-
-- code: |
-    # With initial=1.0 and inc=0.5, we generate the sequence 1.0, 1.5, 2.0, 2.5, 3.0, ...
-    sequence(age, initial=1.0, inc=0.5)
-
-
-- markdown: |
-    ### Inspecting Array objects
-
-
-- code: |
-    # create a test array
-    ndtest([age, gender, time])
-
-
-- markdown: |
-    Get array summary : metadata + dimensions + description of axes + dtype + size in memory
-
-
-- code: |
-    arr.info
-
-
-- markdown: |
-    Get axes
-
-
-- code: |
-    arr.axes
-
-
-- markdown: |
-    Get number of dimensions
-
-
-- code: |
-    arr.ndim
```

Comment:
AFAICT, .ndim, .shape, .size, .memory_used and .dtype do not appear anywhere in the tutorial anymore (only appear in the API reference), which is a shame. I agree these are a bit more advanced attributes which probably shouldn't be in getting started but I think they should appear _somewhere_ in the tutorial. Where exactly, I don't know though.

Reply:
There is still a "Miscellaneous" section. Maybe there ?

Reply:
Yes, that's the best option I can think of.